### PR TITLE
Split out discovery provider into two containers

### PR DESCRIPTION
**Pull request by cheran-senthil**
Tested by running on stage discovery 2

### CODE REVIEW

File: discovery-provider/docker-compose.yml [hunk 1]
```diff
@@ -81,6 +81,38 @@ services:
       - override.env
     environment:
       - audius_discprov_infra_setup=audius-docker-compose
+      - audius_no_workers=false
+      - audius_db_run_migrations=false
+    networks:
+      - discovery-provider-network
+
+  indexer:
+    image: audius/discovery-provider:${TAG:-c54dc44721ad3c8352f0cec56b94002cffe9512a}
```
Comment (joaquincasares) [inline discovery-provider/docker-compose.yml:90]: Let's use `container_name: indexer` so that ELK can grab logs.

**Comment by cheran-senthil** [comment]
https://github.com/AudiusProject/audius-protocol/pull/3853/files

File: discovery-provider/docker-compose.yml (hunk 1 above)
Comment (joaquincasares) [reply discovery-provider/docker-compose.yml:90]: Changes have already been made, built, and pushed to Docker Hub via this commit for the Filebeat container:

* https://github.com/AudiusProject/audius-protocol/pull/3846/commits/c3a1c526e62b482c3be12db8d818de989fabfe5e

**Review by joaquincasares — CHANGES_REQUESTED** [review]

**Review by joaquincasares — APPROVED** [review]

**Review by jowlee — APPROVED** [review]

**Comment by cheran-senthil** [comment]
This stopped working on stage DP2, will merge after figuring this out

**Comment by raymondjacobson** [comment]
@cheran-senthil we can probs merge now right?

**Pull request merged**